src/app/data-explorer/data-element-search-result/data-element-search-result.component.html

```diff
@@ -16,40 +16,42 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/85611988/214616880-6d3b34f1-5a05-46a7-9de6-5f003252a272.png)

Inconsistent spacing on this page where an element does/does not have an identifying badge
(include badge every time but set opacity to 0 maybe?)

Reply:
![image](https://user-images.githubusercontent.com/3219480/214818669-170236b7-3d66-4f27-8934-90e143c1e4cd.png)

Will add this "No description available" to have at least some content. Hopefully the model will contain enough metadata to not require showing this.